examples/dagster/hamilton_code/README.md

```diff
@@ -18,6 +18,12 @@ The directory also include `mock_api.py` which simulates a `ressource` in the Da
     python run.py
     ```
 
+Or run it in Google Colab:
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)
+](https://colab.research.google.com/github/dagworks-inc/hamilton/blob/main/examples/dagster/hamilton_code/notebook.ipynb)
```

Comment:
This one doesn't have a `pip install` and won't work because it rely on local files from the Dagster subdirectory